go/test/endtoend/cluster/vtgate_process.go

```diff
@@ -178,7 +178,7 @@ func (vtgate *VtgateProcess) GetStatusForTabletOfShard(name string, endPointsCou
 // WaitForStatusOfTabletInShard function waits till status of a tablet in shard is 1
 // endPointsCount: how many endpoints to wait for
 func (vtgate *VtgateProcess) WaitForStatusOfTabletInShard(name string, endPointsCount int) error {
-	timeout := time.Now().Add(15 * time.Second)
+	timeout := time.Now().Add(30 * time.Second)
```

Comment:
Why this was increased?

Reply:
Why fail the entire test, which then results in waiting 30 mins if the runner is just super slow? Seems wiser to increase the wait time than fail the test since all we can then do is retry. I can remove this change though if you want.

Reply:
keep it, that makes sense, thank you :)